source/issue-a-credential/credential/index.html.md.erb

```diff
@@ -253,161 +260,82 @@ The example below shows the structure of a verifiable credential using a JSON We
-
-Below is an example of a credential error response when the credential request included an invalid proof of possession:
+#### Signature  
 
-```
-HTTP/1.1 400 Bad Request
-Content-Type: application/json
-Cache-Control: no-store
-
-{
-  "error": "invalid_proof"
-}
-```
+The credential must be signed with your credential issuer’s private signing key using the Elliptic Curve Digital Signature Algorithm (ECDSA) cryptographic algorithm with P-256 (also known as Secp256r1) elliptic curve.
 
 ### Further guidance on credential binding
 
 #### Binding credentials to users
 
-Because each GOV.UK Wallet instance can be uniquely identified, your service can bind a credential with a specific wallet instance. GOV.UK Wallet uses a specific type of [decentralised identifier (DID)](https://www.w3.org/TR/did-core/) called a `did:key` to cryptographically bind credentials to a user's wallet.
+Because each GOV.UK Wallet instance can be uniquely identified, your service can bind a credential with a specific wallet instance. GOV.UK Wallet uses a specific type of [decentralised identifier (DID)](https://www.w3.org/TR/did-core/) called a `did:key` to cryptographically bind credentials to a user’s wallet.
 
 A [did:key](https://w3c-ccg.github.io/did-method-key/) is a DID method. The DID represents the public key of an asymmetric key pair generated when GOV.UK Wallet is installed on a device. The private key never leaves the device, whereas the `did:key` is shared with credential issuers and verifiers. This allows credentials to be cryptographically bound to a specific GOV.UK Wallet instance.
 
 The GOV.UK Wallet creates a `did:key` from a **P-256** (also known as Secp256r1) elliptic curve public key.
 
-#### The `did:key` format
+#### Using the `did:key` format
 
-The `did:key` method is used to transfer public keys.
+You can use the `did:key` method to transfer public keys.
 
-The format of a `did:key` is `did:key:multibaseValue`. The `multibaseValue` is the base58-btc multibase string representation of concatenating the multicodec identifier for the public key type and the compressed public key.
-
-````
-did-key-format := did:key:MULTIBASE(base58-btc, MULTICODEC(public-key-type, raw-public-key-bytes))
-````
+The format of a `did:key` is `did:key:multibaseValue`. The `multibaseValue` is the multicodec identifier for the public key type followed by the compressed public key, encoded as a base58-btc multibase string.
 
-In Elliptic Curve Cryptography (ECC), the public key is a pair of `x` and `y` coordinates. A compressed public key is the `x` coordinate, which is 32 bytes in length, with a prefix, of 1 byte in length, that indicates whether the `y` coordinate is even or odd. The prefix is `02` if the `y` coordinate is even and `03` if it is odd. The resulting compressed public key is **33 bytes** in length:
+To generate the `multibaseValue`:
```

Comment:
This bit ties into my placeholder comment above. I wonder if this might be better expressed like this?:

```suggestion
To generate the `did:key` value:
```